rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -1161,11 +1167,18 @@ std::vector<std::vector<u8>> get_rpcn_msgs()
 	auto& nc = g_fxo->get<network_context>();
 	{
 		std::lock_guard list_lock(nc.list_p2p_ports_mutex);
-		auto& def_port = nc.list_p2p_ports.at(3658);
+		if (nc.list_p2p_ports.contains(3658))
```

Comment:
If you check for existence of port you may want to change all the .at to [], it's faster.

Reply:
since nt_p2p_port does not have a default constructor, it won't compile for list_p2p_ports[x]